uber/tests/conftest.py

```diff
@@ -184,6 +187,11 @@ def cp_session():
     cherrypy.session = {}
 
 
+@pytest.fixture(autouse=True)
+def reset_threadlocal_cache():
+    threadlocal.clear()
```

Comment:
Ah good catch, I do this in a ton of tests at work but didn't think to put it in here when I was implementing ``@request_cached_property``.